tests/ocf-meta-inf_manifest/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,8 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>Epub with manifest file inside meta-inf</title>
```

Comment:
```suggestion
    <title>Manifest information is ignored</title>
```

My suggestion here relates to my comment on the description in the package — I think this test is testing ignoring *both* manifest.xml and also content.xml, and the statement in question is "Reading systems MUST NOT use ancillary manifest information contained in the ZIP archive or in the [manifest.xml file](https://www.w3.org/TR/epub-33/#sec-container-metainf-manifest.xml)", so I think that's a better title; let me know if I'm misunderstanding (or if you just disagree).